runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java

```diff
@@ -348,6 +363,10 @@ private boolean isTrimCoveredByCheckpoint(long trimMark) {
                 getCurrentContext().checkpoint.startAddress >= trimMark;
     }
 
+    private boolean checkpointCapableStream() {
```

Comment:
It seems that checkpointCapable is a more appropriate name.

Reply:
Done.